seth/gas_adjuster.go

```diff
@@ -235,18 +234,15 @@ func (m *Client) GetSuggestedEIP1559Fees(ctx context.Context, priority string) (
 	}
 
 	if baseFee64 == 0.0 {
-		err = errors.New(ZeroGasSuggestedErr)
-
-		L.Error().
-			Err(err).
+		L.Debug().
 			Float64("BaseFee", baseFee64).
 			Int64("SuggestedTip", currentGasTip.Int64()).
-			Msg("Incorrect gas data received from node. Skipping automation gas estimation")
+			Msgf("Incorrect gas data received from node: historical base fee was 0. Skipping automation gas estimation")
 		return
 	}
 
 	if currentGasTip.Int64() == 0 {
-		L.Warn().
+		L.Debug().
```

Comment:
If it can cause a transaction to hang indefinitely from your experience maybe we better still have it as `Warn`?

Reply:
yup, in that case let me use fallback value from config